app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/view_return.scala.html

```diff
@@ -71,6 +73,67 @@
 @title = @{messages("viewReturn.title")}
 @amountOwed = @{returnSummary.mainReturnChargeAmount}
 
+@glarAmount = @{
```

Comment:
it would take up less space if you just extracted out the glar value where you need it rather than doing a big patter match here - where you need it, a particular type of complete return will be in scope for you to extract it out of

Reply:
fixed